### PR TITLE
#340, Add tuner.pandora.com to `NSExceptionDomains`. 

### PR DESCRIPTION
**Pull request by DanielStormApps**
# Summary

* ... Set `HermesMainWindow` `titlebarAppearsTransparent` to `YES`.

* #340, Resolve Xcode `self` warnings.

* #340, Update to recommended project settings.

# Testing

* Launch `master`, v1.3.1, branch. Receive error:
    * "An error has occurred with Pandora. [Try Again] - Internal Pandora error"

* Launch `fix/#340_SSL-Issue` branch. Receive NO error.

### CODE REVIEW

File: Resources/English.lproj/MainMenu.xib [hunk 1]
```diff
@@ -298,8 +299,8 @@
                 </menuItem>
             </items>
         </menu>
-        <window title="Hermes" allowsToolTipsWhenApplicationIsInactive="NO" autorecalculatesKeyViewLoop="NO" releasedWhenClosed="NO" frameAutosaveName="Preferences" animationBehavior="default" id="371" userLabel="Main Window" customClass="HermesMainWindow">
-            <windowStyleMask key="styleMask" titled="YES" closable="YES" miniaturizable="YES" texturedBackground="YES" unifiedTitleAndToolbar="YES"/>
+        <window title="Hermes" allowsToolTipsWhenApplicationIsInactive="NO" autorecalculatesKeyViewLoop="NO" releasedWhenClosed="NO" frameAutosaveName="Preferences" animationBehavior="default" titlebarAppearsTransparent="YES" id="371" userLabel="Main Window" customClass="HermesMainWindow">
```
Comment (DanielStormApps) [inline Resources/English.lproj/MainMenu.xib]: `unifiedTitleAndToolbar` is deprecated. Setting `titlebarAppearsTransparent` to `YES` achieves the same UI as before.

**Comment by ZevEisenberg** [comment]
Is `self->_foo` the preferred way to fix those things? What about converting them to properties and using `self.foo` syntax?

**Comment by DanielStormApps** [comment]
@ZevEisenberg Could go either way. Was avoiding having to refactor anything outside the scope of the issue. Xcode auto fixes the errors with `self->`, `Editor > Fix All Issues`.

**Comment by nriley** [comment]
Thanks so much for working on this. Running out of time in the weekend but hope to get to it soon.

**Comment by jaredly** [comment]
```
2018-08-23 15:00:44.124412-0600 Hermes[15720:1110099] Pandora.m:823 -[Pandora sendRequest:] https://tuner.pandora.com/services/json/?method=auth.partnerLogin&partner_id=&auth_token=&user_id=
2018-08-23 15:00:44.219398-0600 Hermes[15720:1110099] CFNetwork SSLHandshake failed (-9807)
2018-08-23 15:00:44.219437-0600 Hermes[15720:1110099] TCP Conn 0x60c000162640 SSLHandshake failed (-9807)
```
I still get this error message when applying that patch

File: Resources/English.lproj/MainMenu.xib (hunk 1 above)
Comment (nriley) [reply Resources/English.lproj/MainMenu.xib]: Where do you see this? Even in the 10.14 SDK NSWindow.h I see no deprecation warnings:

```
    /* Specifies a window whose titlebar and toolbar have a unified look - that is, a continuous background. Under the titlebar and toolbar a horizontal separator line will appear.
     */
    NSWindowStyleMaskUnifiedTitleAndToolbar = 1 << 12,
```

**Comment by nriley** [comment]
I'm OK with the `self->` stuff — it's just a syntax change.

**Comment by nriley** [comment]
@jaredly — same here, the ATS exceptions aren't enough to let tuner.pandora.com through.  Going to investigate adding/pinning the bogus certificate.

Also, we sometimes use internal-tuner.pandora.com for Pandora One users. Might be able to remove that as it was redirecting to tuner.pandora.com in my experience.

File: Resources/English.lproj/MainMenu.xib (hunk 1 above)
Comment (nriley) [reply Resources/English.lproj/MainMenu.xib]: OK, I think I see what you meant — NSWindowStyleMaskTexturedBackground is “to be deprecated” and this is what breaks when built with the 10.14 SDK. We may want to draw our own gradient, use some of the fancy new style stuff so dark mode looks awesome, or figure out another way to handle this.

**Pull request closed without merging**